README.md

```diff
@@ -1 +1,54 @@
-# Book-A-Meal_api
\ No newline at end of file
+[![Build Status](https://travis-ci.org/jaodur/Book-A-Meal_api.svg?branch=Develop)](https://travis-ci.org/jaodur/Book-A-Meal_api)
+[![Coverage Status](https://coveralls.io/repos/github/jaodur/Book-A-Meal_api/badge.svg?branch=Develop)](https://coveralls.io/github/jaodur/Book-A-Meal_api?branch=Develop)
+
+## __Book A Meal__ 
+is a web based app that enables  several caterers to setup menus,and manage orders. The app also allows
+customer to order a meal, modify their orders, check their order history and also have a quick glance at the trending
+menus.
+
+The app is built with ___python/Flask___ for its backend.
+
+## __How to setup the api backend__
+
+## ___Prerequisites___
+
+* `Git` [Guide to Git](https://git-scm.com/doc) [Installing Git](https://gist.github.com/derhuerst/1b15ff4652a867391f03)
+* `Python 3.5 or higher`
+* `Pip` [Guide to installing pip](https://github.com/BurntSushi/nfldb/wiki/Python-&-pip-Windows-installation)
+
+
+## __Cloning and installing dependencies__
+* After installing the **`prerequisites`** above, clone the repository Develop branch
+using this command git `clone -b Develop https://github.com/jaodur/Book-A-Meal_api.git`
```

Comment:
This can be changed to `git clone https://github.com/jaodur/Book-A-Meal_api.git`